app/dashboards/import_dashboard.rb

```diff
@@ -0,0 +1,90 @@
+require "administrate/base_dashboard"
+
+class ImportDashboard < Administrate::BaseDashboard
+  # ATTRIBUTE_TYPES
+  # a hash that describes the type of each of the model's fields.
+  #
+  # Each different type represents an Administrate::Field object,
+  # which determines how the attribute is displayed
+  # on pages throughout the dashboard.
+  ATTRIBUTE_TYPES = {
+    id: Field::String,
+    assignee: Field::BelongsTo,
+    type: Field::String,
+    courtesy_notification: Field::Select.with_options(searchable: false, collection: ->(field) { field.resource.class.send(field.attribute.to_s.pluralize).keys }),
+    metadata: Field::String.with_options(searchable: false),
+    file: Field::ActiveStorage,
+    parent: Field::Polymorphic,
+    processed_at: Field::DateTime,
+    processing_errors: Field::Text,
+    public_document: Field::Boolean,
+    status: Field::Select.with_options(searchable: false, collection: ->(field) { field.resource.class.send(field.attribute.to_s.pluralize).keys }),
+    created_at: Field::DateTime,
+    updated_at: Field::DateTime
+  }.freeze
+
+  # COLLECTION_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's index page.
+  #
+  # By default, it's limited to four items to reduce clutter on index pages.
+  # Feel free to add, remove, or rearrange items.
+  COLLECTION_ATTRIBUTES = %i[
+    created_at
+    type
+    file
+    assignee
+    public_document
+  ].freeze
+
+  # SHOW_PAGE_ATTRIBUTES
+  # an array of attributes that will be displayed on the model's show page.
+  SHOW_PAGE_ATTRIBUTES = %i[
+    id
+    file
+    assignee
+    courtesy_notification
+    metadata
+    parent
+    processed_at
+    processing_errors
+    public_document
+    status
+    type
```

Comment:
I think this should be more prominent, perhaps after the `id`. What do you think?

Reply:
fixed in 1101286e8773b16f52c65e49faeb401917a10cbd